src/cmd/ksh93/edit/edit.c

```diff
@@ -590,7 +590,10 @@ void ed_setup(Edit_t *ep, int fd, int reedit) {
             if (r) sh_offoption(shp, SH_RESTRICTED);
             sh_trap(shp, ".sh.subscript=$(tput cuu1 2>/dev/null)", 0);
             pp = nv_getval(SH_SUBSCRNOD);
-            if (pp) strncpy(CURSOR_UP, pp, sizeof(CURSOR_UP) - 1);
+            if (pp) {
+                // It should be impossible for the cursor up string to be truncated.
+                if (strlcpy(CURSOR_UP, pp, sizeof(CURSOR_UP)) >= sizeof(CURSOR_UP)) abort();
```

Comment:
`sizeof(CURSOR_UP)) >= sizeof(CURSOR_UP)`

This will always evaluate to `1`. What is the point of this change ?

Reply:
Sorry I missed the closing parenthesis.